Update on timeline for MV2 sunset (#2237)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

This timeline is subject to change. This article will be updated to share exact details as the milestones draw nearer.

> We acknowledge that Chrome has revised the timelines for Manifest V2 sunset. We are in the process of updating this document. Watch this space for more updates.

Author: @mikehoffms, we added a short note about working on revised timelines for MV2 sunset, given Chrome has provided new timelines.

Could you please suggest how we could make this more prominent somehow? Would love to get your inputs on the content as well.

CC: @ashishnirvana

Reviewer: re: prominent: this article, though short, needs subheadings.  Skip odd headings; use h2/h4/h6 due to current .css.

Even though dup's the title, this list can/should be introduced by an h4 or h2 heading:
h4 Overview of changes from V2 to V3  (my long-term PR adds a comparable table of changes, see url tbd)
An overview of the changes involved, ... 

You could group the new sentence and maybe some other "Oct 2020" and "June 2023" sentences under a h4 or h2 heading like "Revised timeline"

Misc suggestions:
Change:
Support for JavaScript promises will be added to many methods.
to:
Promise support will be added to many methods.

Despite Chrome style guide Timeless Docn page, in this case I recommend changing from:
Microsof Edge stops running Manifest V2 ...
to:
Microsof Edge will stop running Manifest V2 ...

I'd try to gather some similar sentences into a paragraph and be more terse reducing wordcount where the messaging is maybe repeated.  Maybe delete or shorten "Microsoft continues to iteratively improve the platform and address the feedback shared by extension developers." - somewhat an inherent & implied given.  It could be converted to a request for input.

Reviewer 2: Resolving based on Michael's instructions.

> This timeline is subject to change. This article will be updated to share exact details as the milestones draw nearer.
> 
> We acknowledge that Chrome has revised the timelines for Manifest V2 sunset. We are in the process of updating this document. Watch this space for more updates.

Reviewer: ```suggestion

#### Revised timeline

This timeline is subject to change. This article will be updated as the milestones draw nearer.  Chrome has revised the timeline for Manifest V2 sunset, and we are about to update this article to reflect that change.
```
This Suggestion reduces wordcount to focus the communication, and creates a section by adding a small heading.  I plan to create a separate PR containing a general structural edit of this article, to enable this PR to be merged now.
@Reezaali  @sonakshimore

Reviewer 2: Committing based on discussion with Michael.
